### PR TITLE
Fix button icons alignment

### PR DESCRIPTION
**Pull request by anserwaseem**
## Describe your changes
1. Fix button icons alignment
2. Override default styles of Lottie when given from yaml

### Screenshots [Optional]
1.
Before:
image
After:
image

2. for same styles of Lottie:
Before:
image
After:
Screenshot 2024-05-13 at 2 29 01 PM

## Issue ticket number and link

Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added tests
- [x] I have added a changeset `pnpm changeset add`
- [x] I have added example usage in the kitchen sink app

### CODE REVIEW

File: packages/runtime/src/shared/styles.ts [hunk 1]
```diff
@@ -104,13 +104,21 @@ export const getIcon = (name: string): SvgIconComponent | undefined => {
 export const getComponentStyles = (
   name: string,
   styles?: React.CSSProperties,
-  returnAsString = true,
+  options: {
+    importantStyles?:
+      | keyof React.CSSProperties
+      | Array<keyof React.CSSProperties>;
+    returnAsString?: boolean;
```
Comment (evshi) [inline packages/runtime/src/shared/styles.ts:111]: Why are these needed?

File: packages/runtime/src/widgets/Button.tsx [hunk 2]
```diff
@@ -74,7 +74,13 @@ export const Button: React.FC<ButtonProps> = ({ id, onTap, ...rest }) => {
         }}
       >
         {!loading && (
-          <>
+          <div
+            style={{
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "center",
```
Comment (evshi) [inline packages/runtime/src/widgets/Button.tsx:81]: Hmm this will break how styling will work by inserting another child element. These styles are already on the parent element - did this regress?

File: packages/runtime/src/shared/styles.ts (hunk 1 above)
Comment (anserwaseem) [reply packages/runtime/src/shared/styles.ts:111]: to override these styles of svg
image

File: packages/runtime/src/widgets/Button.tsx (hunk 2 above)
Comment (anserwaseem) [reply packages/runtime/src/widgets/Button.tsx:81]: Yea. The label and icons are not aligned since this [PR](https://github.com/EnsembleUI/ensemble-react/pull/572) there's a span containerizing the icons and label, we need to align them

Comment (evshi) [reply packages/runtime/src/widgets/Button.tsx:81]: Weird I didn't notice while testing. Not sure how this fixes the issue if that span is still wrapping the icons and label?

File: packages/runtime/src/shared/styles.ts (hunk 1 above)
Comment (evshi) [reply packages/runtime/src/shared/styles.ts:111]: Can you provide more detail about the problem? Which styles specifically?

You should be able to achieve what you're looking for without these extra properties.
